go.mod

```diff
@@ -6,11 +6,11 @@ require (
 	github.com/gin-gonic/gin v1.9.1
 	github.com/google/uuid v1.6.0
 	github.com/prometheus/client_golang v1.12.1
-	github.com/stretchr/testify v1.9.0
+	github.com/stretchr/testify v1.10.0
 	github.com/uber-go/tally/v4 v4.1.1
 	github.com/uber/cadence-idl v0.0.0-20220713235846-fda89e95df1e
 	github.com/urfave/cli v1.22.5
-	go.temporal.io/sdk v1.30.0
+	go.temporal.io/sdk v1.33.1
```

Comment:
Suggested by temporal team to upgrade to 1.33 otherwise API key may need extra code for namespace header
<img width="537" alt="Screenshot 2025-04-17 at 11 14 44 AM" src="https://github.com/user-attachments/assets/5a986bb2-63e8-4202-b65a-5067cc31dbcf" />